superset-frontend/plugins/legacy-preset-chart-deckgl/src/layers/Polygon/Polygon.jsx

```diff
@@ -57,7 +58,7 @@ function setTooltipContent(formData) {
     return (
       <div className="deckgl-tooltip">
         {o.object.name && (
-          <TooltipRow label="name: " value={`${o.object.name}`} />
+          <TooltipRow label={t('name: ')} value={`${o.object.name}`} />
```

Comment:
Another example of what I commented on elsewhere. I would think `name` is already translated/used in numerous places, so translating things like this with punctuation/spacing kind of adds some overhead.

Reply:
Totally agree